docs/submission/viewing-clinical-data.md

```diff
@@ -0,0 +1,99 @@
+---
+id: viewing-clinical-data
+title: Viewing Clinical Data
+platform_key: DOCS_VIEWING_CLINICAL_DATA
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The submitted data page is where you can review, query, and download clinical data submitted by your program. Submitted clinical data will be accessible from this page following sample registration and clinical data submission sign-off. Only program administrators, data submitters, and collaborators can access the program's submitted data page.
+
+## Navigating the Submitted Data Page
+
+The submitted data dashboard is accessed by clicking the "submitted data" tab on the left-hand menu. The dashboard has two main sections, the **donor filter pane** and the **clinical data table pane**.
```

Comment:
Should we capitalize "Donor Filter" pane" and "Clinical Data Table" pane, so it matches the diagram?

Reply:
I think it's fine because it's referring to the actual platform-ui and not to the diagram.

but it's a quick change if you want me to make it